vmck/api.py

```diff
@@ -23,14 +32,42 @@ def home(request):
     })
 
 
-def create_job(request):
-    options = json.loads(request.body) if request.body else {}  # TODO validate
-
+def process_options(options):
     options.setdefault('cpus', 1)
     options.setdefault('memory', 512)
     options.setdefault('image_path', 'imgbuild-master.qcow2.tar.gz')
     options.setdefault('name', 'default')
-    options['cpu_mhz'] = options['cpus'] * settings.QEMU_CPU_MHZ
+    options['cpu_mhz'] = int(options['cpus']) * settings.QEMU_CPU_MHZ
```

Comment:
The user should always send `cpus` as an integer, it's not our job to convert their broken json :)